drivers/edac/shell.c

```diff
@@ -11,6 +11,8 @@
 #include <drivers/edac.h>
 #include "ibecc.h"
 
+#define DEVICE_NAME DT_LABEL(DT_NODELABEL(ibecc))
```

Comment:
again, why use NODELABEL here instead of COMPAT?

Reply:
I am not fluent in DTS programming language, just followed User Guide Option2: https://docs.zephyrproject.org/latest/guides/dts/howtos.html#option-2-create-devices-using-node-labels